addons/web/static/src/core/popover/popover.scss

```diff
@@ -1,153 +1,97 @@
 /*!rtl:begin:ignore*/
-@keyframes slide-top {
-  0% {
-    opacity: 0;
-    transform: translateY(-5%);
-  }
 
-  100% {
-    opacity: 1;
-    transform: translateY(0);
-  }
+@keyframes slide-top {
```

Comment:
To keep in mind: we'll have to move animations from all components to a specific file.
I think that currently we have several repetition